src/main/resources/application.yaml

```diff
@@ -344,6 +346,9 @@ govnotify:
         email: 0df260d7-fd82-430b-86bb-047083a8172a
       detentionEngagementTeam:
         email: c15c1d5d-aa44-4725-b953-14bd506cf7ce
+    generateHearingBundle:
+      detentionEngagementTeam:
```

Comment:
Do we need a new template here? Cant we re-use the one which was created as a part of RIA-6782, as that work is anyway redundant now. Make sure that template is not being used anywhere else.

Reply:
The handler conditions for the notification created for RIA-6782 is different
![image](https://github.com/hmcts/ia-case-notifications-api/assets/118450580/2fc9debb-ead5-40ec-827e-63f7c817395c)

Although you are correct in that I could have reused the same personalisation. Shall I refactor to use same personalisation or leave as is now that it is done?

Reply:
In that case we basically need to take out the redundant classes as a part of this ticket itself. Otherwise it will add on to the tech-debt.
Also, just completion of GENERAT_HEARING_BUNDLE doesn't ensure that hearing bundle is ready. I think the same logic is used in the LR journey as well. Please check that as well.